forum/settings.py

```diff
@@ -12,6 +12,7 @@
 
 from pathlib import Path
 import os
```

Comment:
Is module os required in settings? Maybe it is sense to remove it?

Reply:
Fixed